setup.py

```diff
@@ -1,3 +1,4 @@
 from setuptools import setup
 
-setup()
+
+setup(use_scm_version=True)
```

Comment:
It seems that only one of `setup.{py,cfg}` or `pyproject.toml` is sufficient to set `use_scm_version`.
It would be nice if only one of them is set up, because it is difficult to understand if both are there.

ref: https://github.com/pypa/setuptools_scm
> Including this section is comparable to supplying use_scm_version=True

Reply:
Thx!